server/controllers/users.js

```diff
@@ -39,11 +39,51 @@ const deleteUsers = async (req, res, next) => {
     }
 }
 
-// Exports
+const getOneuser = async (req, res, next) => {
```

Comment:
is there a reason this is not fully camelCase "getOneuser" ,and the others follow camel case like "updateOneUser"?

Reply:
It just define a variable using camel case for getOneUser and updateOneuser. you can give any name you want.